modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/api.mustache

```diff
@@ -52,47 +52,34 @@ class {{classname}} (
         {{^bodyParam}}
         val body: Any? = null
         {{/bodyParam}}
-        {{#allParams}}
-        {{#required}}
-        // verify the required parameter '{{paramName}}' is set
-        // This is probably taken care of by non-null types anyway
-        requireNotNull({{paramName}})
-        {{/required}}
-        {{/allParams}}
```

Comment:
The Kotlin type systems makes sure that the parameters cannot be null. Completely redundant check.